dashboard/app/serializers/api/v1/pd/workshop_serializer.rb

```diff
@@ -63,6 +63,10 @@ def enrollment_code
     @scope.try(:[], :enrollment_code)
   end
 
+  def attended
+    object.enrollments.find_by(code: @scope.try(:[], :enrollment_code))&.attendances&.any?
```

Comment:
Whereas here we need the two safe-navigation operators because `find_by` might have returned `nil`.